src/locales/en-US.json

```diff
@@ -204,6 +206,24 @@
     "userManagement": "User management",
     "virtualMedia": "Virtual media"
   },
+  "pageAuditLogs": {
+    "table": {
+      "id": "ID",
+      "op": "Operation",
+      "acct": "Account",
+      "addr": "Address",
+      "res": "Response",
+      "messageArgs": "MessageArgs",
+      "message": "Message",
+      "searchLogs": "Search logs",
+      "eventTimeStamp": "Date"
+    },
+    "toast": {
+      "infoStartDownload": "Download might take several minutes to complete.",
+      "errorStartDownload": "Error downloading the Additional data.",
```

Comment:
I think we have to rephrase this: "Error downloading Logs"

Reply:
Addressed